src/test/util_string_tests.cpp

```diff
@@ -11,18 +11,30 @@ using namespace util;
 
 BOOST_AUTO_TEST_SUITE(util_string_tests)
 
+template <unsigned NumArgs>
+std::string TfmFormatZeroes(const char* fmt)
```

Comment:
Is it coincidental that every single format type accepts `0` as a valid substitution?

Reply:
So %n is the only one that doesn't accept zeroes, right?

Reply:
Have not fuzzed tinyformat but that seems like a good guess.